packages/firebase_core/firebase_core_platform_interface/test/platform_interface_tests/platform_interface_firebase_core_test.dart

```diff
@@ -35,21 +35,30 @@ void main() {
 
 class ImplementsFirebasePlatform implements FirebasePlatform {
   @override
-  Future<FirebaseAppPlatform> initializeApp(
-          {String name, FirebaseOptions options}) =>
-      null;
+  Future<FirebaseAppPlatform> initializeApp({
+    String? name,
+    FirebaseOptions? options,
+  }) async {
+    return FakeFirebaseAppPlatform();
+  }
 
   @override
   FirebaseAppPlatform app([String name = defaultFirebaseAppName]) {
-    return null;
+    return FakeFirebaseAppPlatform();
   }
 
   @override
-  List<FirebaseAppPlatform> get apps => null;
+  List<FirebaseAppPlatform> get apps => [];
```

Comment:
These changes were necessary to make the tests compile. Afaik this doesn't matter but it'd be worth double checking that this doesn't change the behaviour of the related tests

Reply:
This change looks ok to me